eo-maven-plugin/src/main/java/org/eolang/maven/DcsWithRuntime.java

```diff
@@ -23,47 +23,107 @@
  */
 package org.eolang.maven;
 
+import com.jcabi.xml.XMLDocument;
+import java.io.IOException;
+import java.net.URL;
 import java.util.Iterator;
 import org.apache.maven.model.Dependency;
 import org.cactoos.list.ListOf;
+import org.cactoos.scalar.Sticky;
+import org.cactoos.scalar.Unchecked;
 
 /**
  * Add runtime dependency if it is absent.
  *
  * @since 0.28.11
- * @todo #1361:90min Hardcoded version of EoRuntimeDependency.
- *  See the EoRuntimeDependency constructor for more info.
- *  It's much better to determine the version of the runtime library
- *  dynamically. For example, we can fetch the latest version by http
- *  or from config files.
  */
 final class DcsWithRuntime implements Dependencies {
 
     /**
-     * The dependencies source.
+     * Dependency downloaded by HTTP from Maven Central.
+     */
+    private static final Unchecked<Dependency> FROM_MAVEN = fromMaven();
```

Comment:
@volodya-lombrozo let's rename to `dynamic`. Also static methods should be called via class name. 
So this would look like this:
```private static final Unchecked<Dependency> DYNAMIC = DcsWithRuntime.download()```

Reply:
@volodya-lombrozo Actually you can use [Sticky](https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/scalar/Sticky.java) here from Cactoos lib and get rid of static method.

Reply:
@mximp I'm already using `Sticky` class:
```java
 private static Unchecked<Dependency> dependency(final String version) {
        return new Unchecked<>(
            new Sticky<>(
                () -> {
                    final Dependency dependency = new Dependency();
                    dependency.setGroupId("org.eolang");
                    dependency.setArtifactId("eo-runtime");
                    dependency.setVersion(version);
                    dependency.setClassifier("");
                    return dependency;
                }
            )
        );
    }
```
I can't declare field as `Sticky`, because in that case I have to deal with `Exception` handling.

Reply:
@mximp also I can't get rid of the static method, because I wanna leave `private static final Unchecked<Dependency> FROM_MAVEN ` as a static field in order to increase speed of the tests. (when we have a static field, the method will be called only ~2 times, if you make it as a simple object field  - the download will be happened each time when you create a new `DcsWithRuntime` object > 10 times)

Reply:
@volodya-lombrozo I just mean why do we need the method `dependency`. It just creates an object and used only in one place - we can inline it:
```
private static final Unchecked<Dependency> MAVEN_DEPENDENCY = new Unchecked<>(
    new Sticky<>(() -> {
        final Dependency dependency = new Dependency();
        dependency.setGroupId("org.eolang");
        dependency.setArtifactId("eo-runtime");
        dependency.setVersion(version);
        dependency.setClassifier("");
        return dependency;
    })
);
```

Reply:
@volodya-lombrozo corrected inlining:
```
private static final Unchecked<Dependency> MAVEN_DEPENDENCY =
    new Unchecked<>(
        new Sticky<>(() -> {
            final Dependency dependency = new Dependency();
            dependency.setGroupId("org.eolang");
            dependency.setArtifactId("eo-runtime");
            dependency.setVersion(
                new XMLDocument(
                    new URL(
                        String.format(
                            "https://repo.maven.apache.org/maven2/%s/maven-metadata.xml",
                            "org/eolang/eo-runtime"
                        )
                    )
                ).xpath("//latest/text()").get(0)
            );
            dependency.setClassifier("");
            return dependency;
        })
    );
```

Reply:
@mximp I got your idea. It has several advantages:
- we don't create extra methods
- we reduce total number of lines

On the other hand, that approach:
- creates lots of code that is placed above fields declarations and constructors (it's confusing to see a lot of code even before declaring variables).
- looks like a solid pease of code which do several things at once: (1) Creates dependency (2) Downloads latest version of library from maven. (3) Creates `Sticky` object. (some sort of violation of the principle of single responsibility)

I prefer using small methods and classes for achieving granularity and better code readability. But I can be mistaken, of course. Especially in "better code readability".

Reply:
@volodya-lombrozo I believe the biggest advantage of the inlined version is that I can see the whole logic of defining `MAVEN_DEPENDENCY` in front of me: that it's sticky (i.e. cached), which URL used and that latest version is extracted. It's only several lines of code and I don't need to jump around the methods.

Regarding SRP, in this case I think we need to look at it a bit broader: we just extract version from Maven Central. I would probably agree that dealing with URL and parsing XML for a version might be a distinguishable bit of functionality but in this case making it to be a separate method doesn't provide any benefits: we just spread logic across multiple places.

Reply:
@yegor256 what is the better way?